_posts/2022-11-07-changelog-182.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #182"
+date: 2022-11-07 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8755] Support `let_chains` (by [@Kobzol])
+
+* [#9105] Allow to run a specific [documentation test](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html) [all the ways](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-testing.html) you can run a usual test (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-182/run_doctest" w="700px" %}
```

Comment:
It would be nice to show Test tool window here.
Maybe drop one doc test from the example and show Test tool window instead?